### PR TITLE
Working days (Independent Publisher)

### PR DESCRIPTION
**Pull request by tposzytek**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [X] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Connector validation

Cloud flow validation

Paconn validation

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hello @tposzytek,
Thank you very much and congratulations on your new submission. Our Certification Engineers will review it within 1-2 weeks.
Please let us know if you have any questions.
Thank you very much again for working with us.

File: independent-publisher-connectors/Working days/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2104 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.2.0",
+    "title": "Working days",
+    "description": "Working days API. It covers more than 45 countries and 230 regional calendars. It also covers main stock exchanges calendars. The quality of data is the top priority. API offers up to date information by following government announcements regarding public holidays changes.",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Working days/apiDefinition.swagger.json:6]: @tposzytek word `API` is not allowed in the description, please remove it. The description itself should reflect the main value and purpose offered by the connector. More information about the description can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

File: independent-publisher-connectors/Working days/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2104 @@
...
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/1.2/add_working_days": {
+      "get": {
+        "summary": "Add working days",
+        "description": "Add (or remove) any number of working days to a date.",
+        "operationId": "Add_working_days",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Working days/apiDefinition.swagger.json:43]: @tposzytek please use pascal case for operationId properties. More information can be found in [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers).

File: independent-publisher-connectors/Working days/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2104 @@
...
+                          "type": "string",
+                          "description": "Weekend days taking place on Sundays within the period",
+                          "title": "All weekend days on Sundays"
+                        }
+                      }
+                    },
+                    "public_holidays": {
+                      "title": "Public holidays",
+                      "type": "object",
+                      "description": "public_holidays"
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Working days/apiDefinition.swagger.json:725]: @tposzytek descriptions should be valid sentences. Please refer to [English strings recommendations](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @tposzytek,

Please check comments after the review.

Let us know if you have any questions.

File: independent-publisher-connectors/Working days/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Tomasz Poszytek",
+    "stackOwner": "api.workingdays.org"
```
[the remark this answers is not in the document]
Comment (tposzytek) [reply independent-publisher-connectors/Working days/apiProperties.json:21]: There is not much written on the site. I know a person who is the developer behind the project, but don't want them to get listed here. Is "Workingdays API" enough? This name is present in all terms of use and cookies policy pages.

File: independent-publisher-connectors/Working days/apiDefinition.swagger.json (hunk 3 above)
Comment (tposzytek) [reply independent-publisher-connectors/Working days/apiDefinition.swagger.json:725]: Checked everywhere and fixed.

File: independent-publisher-connectors/Working days/apiDefinition.swagger.json (hunk 2 above)
Comment (tposzytek) [reply independent-publisher-connectors/Working days/apiDefinition.swagger.json:43]: Checked everywhere and fixed.

**Review by tposzytek — COMMENTED** [review]
All changes mentioned in comments implemented.

**Comment by v-EgorKozhadei** [comment]
[[certify-connector]]

**Comment by v-EgorKozhadei** [comment]
Hello @tposzytek, 

I hope you are doing well. 

Congratulations, your connector is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.  

Please let us know if you have any questions. 

Thank you very much for working with us.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@tposzytek Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**